packages/cli/src/commands/gsn.ts

```diff
@@ -8,6 +8,7 @@ commander
   .command('start', 'all-on-one: deploy all contracts, start relay')
   .command('deploy', 'deploy RelayHub and other GSN contracts instances')
   .command('relayer-register', 'stake for a relayer and fund it')
+  .command('relayer-withdraw', 'withdraw manager hub balance to owner')
```

Comment:
`"Withdraw relayer's manager balance from RelayHub to owner."`

(users know its a relayer, the terms "maanger", "worker" are mostly internal)

Reply:
Ok